iceoryx_platform/cmake/IceoryxPackageHelper.cmake

```diff
@@ -207,7 +207,7 @@ Macro(iox_add_executable)
     if ( QNX )
         target_link_libraries(${IOX_TARGET} ${IOX_LIBS_QNX})
     elseif ( LINUX )
-        target_link_libraries(${IOX_TARGET} ${IOX_LIBS_LINUX})
+        target_link_libraries(${IOX_TARGET} ${IOX_LIBS_LINUX} systemd)
```

Comment:
Please add this only to the iox-roudi target

Reply:
@elBoberido fixed